.gitignore

```diff
@@ -0,0 +1,138 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
```

Comment:
Quite a few pycache files are showing up in the PR - this might need tweaking to manually add `*.pyc`